include/libtorrent/hash_picker.hpp

```diff
@@ -198,18 +198,35 @@ namespace libtorrent
 		file_storage const& m_files;
 		aux::vector<aux::vector<sha256_hash>, file_index_t>& m_merkle_trees;
 		aux::vector<aux::vector<bool>, file_index_t> m_hash_verified;
+
+		// information about every 512-piece span of each file. We request 512
+		// blocks at a time
```

Comment:
This wording is ambiguous since `block` is commonly used to refer to a 16 KB block of piece data. Suggest clarifying that we request 512 blocks of piece hashes at a time.

Reply:
oh, right. at first I thought the hash_picker requested 512 *blocks* at a time, but then I realized it's 512 *pieces* at a time (I think my understanding is right at least). I failed to update this comment correctly